apps/schema/src/components/NomnomDiagram.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <InputCheckbox
+      id="showAttributes"
+      v-model="showAttributes"
+      :defaultValue="showAttributes"
+      :options="['attributes', 'external']"
+    />
+    <div v-html="nomnomlSVG" class="bg-white" style="max-width: 100%"></div>
+  </div>
+</template>
+
+<script>
+import { renderSvg } from "nomnoml";
+import { InputCheckbox } from "molgenis-components";
+
+export default {
+  props: {
+    schema: {
+      type: Object,
+      required: true,
+    },
+  },
+  components: {
+    InputCheckbox,
+  },
+  data() {
+    return {
+      imgFullscreen: false,
+      showAttributes: [],
```

Comment:
the name showAttributes suggests a boolean. suggestion: 'attributesToShow'

Reply:
renamed to 'displaySettings'